### PR TITLE
Pegas

### PR DESCRIPTION
**Pull request by liviurotiul**
PR for an NGS bacteria analysis pipeline called "pegas".
This is my first attempt at an open source contribution so please be patient. :)

### CODE REVIEW

**Comment by liviurotiul** [comment]
@antunderwood @pontus @brentp @ryanraaum 
Hello, sorry to buzz you guys like this, this is my first conda contribution. What are the next steps I need to take to get this approved.
Thanks!

File: recipes/pegas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    - snakemake >=7.32.4
+    - beautifulsoup4 >=4.12.3
+
+test:
+  commands:
+    - {{ name }} --help
+
+about:
+  home: https://github.com/liviurotiul/PeGAS
+  license: GPL-2.0
```
Comment (martin-g) [inline recipes/pegas/meta.yaml:41]: Please use id from https://spdx.org/licenses/

**Review by martin-g — APPROVED** [review]

**Comment by martin-g** [comment]
> What are the next steps I need to take to get this approved.

You just need to mark your PRs with `@BiocondaBot please add label` as explained at https://github.com/bioconda/bioconda-recipes/blob/master/.github/PULL_REQUEST_TEMPLATE.md

**Comment by martin-g** [comment]
@BiocondaBot please add label

**Pull request merged**